### PR TITLE
Update Magento_AdminNotification module ReadMe

### PR DESCRIPTION
**Pull request by diazwatson**
### Description (*)

Updates ReadMe for the Magento_AdminNotification module.

### Fixed Issues (if relevant)

1. magento/devdocs https://github.com/magento/devdocs/issues/4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-5617](https://jira.corp.magento.com/browse/ENGCOM-5617) has been created to process this Pull Request
:eight_spoked_asterisk: @ihor-sviziev, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
QA not applicable

File: app/code/Magento/AdminNotification/README.md [hunk 1]
```diff
@@ -1,4 +1,29 @@
-# Admin Notification
+# Magento_AdminNotification module
 
-**Admin Notification** provides the ability to alert administrators via
-system messages and provides a message inbox for surveys and notifications.
+The Magento_AdminNotification module provides the ability to alert administrators via system messages and provides a message inbox for surveys and notifications.
+
+## Installation details
+
+Before disable or uninstall this module, please note that Magento_Indexer module depends on this module.
```
Comment (jeff-matthews) [inline app/code/Magento/AdminNotification/README.md]:
```suggestion
Before disabling or uninstalling this module, note that the Magento_Indexer module depends on this module.
```

File: app/code/Magento/AdminNotification/README.md [hunk 2]
```diff
@@ -1,4 +1,29 @@
...
+
+For information about a module installation in Magento 2, see [Enable or disable modules](http://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```
Comment (jeff-matthews) [inline app/code/Magento/AdminNotification/README.md]:
```suggestion
For information about module installation in Magento 2, see [Enable or disable modules](http://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```

File: app/code/Magento/AdminNotification/README.md [hunk 3]
```diff
@@ -1,4 +1,29 @@
...
+
+ - `controller_action_predispatch` event in `Magento\AdminNotification\Observer\PredispatchAdminActionControllerObserver`
+
+### Layouts
+
+This module introduces the following layouts handle in the `view/adminhtml/layout` directory:
+
+- `adminhtml_notification_index`
+
+For more information about a layout in Magento 2, see the [Layout documentation](http://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-overview.html).
```
Comment (jeff-matthews) [inline app/code/Magento/AdminNotification/README.md]:
```suggestion
For more information about layouts in Magento 2, see the [Layout documentation](http://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-overview.html).
```

File: app/code/Magento/AdminNotification/README.md [hunk 4]
```diff
@@ -1,4 +1,29 @@
...
+
+### UI components
+
+You can extend admin notifications using the configuration file in `view/adminhtml/ui_component/notification_area.xml`
```
Comment (jeff-matthews) [inline app/code/Magento/AdminNotification/README.md]:
```suggestion
You can extend admin notifications using the `view/adminhtml/ui_component/notification_area.xml` configuration file.
```

File: app/code/Magento/AdminNotification/README.md [hunk 5]
```diff
@@ -1,4 +1,29 @@
...
+
+For information about a UI component in Magento 2, see [Overview of UI components](http://devdocs.magento.com/guides/v2.3/ui_comp_guide/bk-ui_comps.html).
```
Comment (jeff-matthews) [inline app/code/Magento/AdminNotification/README.md]:
```suggestion
For information about UI components in Magento 2, see [Overview of UI components](http://devdocs.magento.com/guides/v2.3/ui_comp_guide/bk-ui_comps.html).
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Please see minor editorial suggestions.

**Review by jeff-matthews — APPROVED** [review]
Thanks @diazwatson!

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-5617](https://jira.corp.magento.com/browse/ENGCOM-5617) has been created to process this Pull Request

**Comment by diazwatson** [comment]
Hi @jeff-matthews, thank for taking the time to review this PR.

File: app/code/Magento/AdminNotification/README.md [hunk 6]
```diff
@@ -1,4 +1,30 @@
...
 
-**Admin Notification** provides the ability to alert administrators via
-system messages and provides a message inbox for surveys and notifications.
+The Magento_AdminNotification module provides the ability to alert administrators via system messages and provides a message inbox for surveys and notifications.
+
+## Installation details
+
+Before disabling or uninstalling this module, note that the Magento_Indexer module depends on this module.
+
+For information about module installation in Magento 2, see [Enable or disable modules](http://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-subcommands-enable.html).
```
Comment (dshevtsov) [inline app/code/Magento/AdminNotification/README.md:9]: Change the link to `https://...`

File: app/code/Magento/AdminNotification/README.md [hunk 7]
```diff
@@ -1,4 +1,30 @@
...
+ - `controller_action_predispatch` event in `Magento\AdminNotification\Observer\PredispatchAdminActionControllerObserver`
+
+### Layouts
+
+This module introduces the following layouts and layout handles in the `view/adminhtml/layout` directory:
+
+- `adminhtml_notification_index`
+- `adminhtml_notification_block`
+
+For more information about layouts in Magento 2, see the [Layout documentation](http://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/layout-overview.html).
```
Comment (dshevtsov) [inline app/code/Magento/AdminNotification/README.md:24]: Change the link to `https://...`

File: app/code/Magento/AdminNotification/README.md [hunk 8]
```diff
@@ -1,4 +1,30 @@
...
+
+### UI components
+
+You can extend admin notifications using the `view/adminhtml/ui_component/notification_area.xml` configuration file.
+
+For information about UI components in Magento 2, see [Overview of UI components](http://devdocs.magento.com/guides/v2.3/ui_comp_guide/bk-ui_comps.html).
```
Comment (dshevtsov) [inline app/code/Magento/AdminNotification/README.md:30]: Change the link to `https://...`

**Review by dshevtsov — CHANGES_REQUESTED** [review]

**Pull request merged**